bindings/python/triplestore/triplestore.py

```diff
@@ -203,6 +203,8 @@ def __add__(self, other):
 DCAT = Namespace("http://www.w3.org/ns/dcat#")
 TIME = Namespace("http://www.w3.org/2006/time#")
 FNO = Namespace("https://w3id.org/function/ontology#")
+QUDTU = Namespace("http://qudt.org/vocab/unit/")
```

Comment:
This is fine, however QUDT uses the prefix _unit_:
 @prefix unit: <http://qudt.org/vocab/unit/> .

Reply:
Maybe it is better that we also uses `UNIT` as prefix?